### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by des-des**
Initial setup new phoenix stack

### CODE REVIEW

**Comment by des-des** [comment]
@nelsonic I think codecov will be doing weird things until we have this in.

File: README.md [hunk 1]
```diff
@@ -1,165 +1,20 @@
-# tudo
+# Tudo
```
Comment (nelsonic) [inline README.md:1]: we should _really_ be re-using the _old_ readme which has/had a lot more introductory detail.

File: package.json [hunk 2]
```diff
@@ -1,69 +1,20 @@
 {
-  "name": "tudo",
-  "version": "1.0.2",
-  "description": "A single version of the truth with integrated task tracking, delegation and accountability.",
-  "main": "index.js",
+  "repository": {},
```
Comment (nelsonic) [inline package.json:2]: we should copy _most_ of the `metadata` from the "old" `package.json` to here (_not the `scripts` or `dependencies`_)

File: web/router.ex [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    plug :fetch_flash
+    plug :protect_from_forgery
+    plug :put_secure_browser_headers
+  end
+
+  pipeline :api do
+    plug :accepts, ["json"]
+  end
+
+  scope "/", Tudo do
```
Comment (nelsonic) [inline web/router.ex:16]: are there any _other_ routes?

**Review by nelsonic — APPROVED** [review]
@des-des the code looks good (_mostly phoenix boilerplate_) the "_good stuff_" will be on future PRs. 👍

**Pull request merged**